src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -9,34 +9,43 @@ import {
   ContentChildren,
   Inject,
   QueryList,
+  OnDestroy,
   ViewContainerRef,
   ViewChild,
+  PLATFORM_ID,
 } from '@angular/core';
 
 import { IfActiveService } from '../../utils/conditional/if-active.service';
 import { IfOpenService } from '../../utils/conditional/if-open.service';
 
 import { TabsService } from './providers/tabs.service';
 import { ClrTabLink } from './tab-link.directive';
-import { ClrTabContent } from './tab-content';
 import { TABS_ID, TABS_ID_PROVIDER } from './tabs-id.provider';
 import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
+import { Subscription } from 'rxjs';
+import { ClrKeyFocus } from '../../utils/focus/key-focus/key-focus';
+import { startWith, filter } from 'rxjs/operators';
+import { isPlatformBrowser } from '@angular/common';
+import { ClrTab } from './tab';
 
 @Component({
   selector: 'clr-tabs',
   template: `
-        <ul class="nav" role="tablist" [attr.aria-owns]="tabIds">
+        <ul class="nav" role="tablist" [attr.aria-owns]="tabIds" [clrKeyFocus]="tabLinkElements" clrDirection="both"
+            (clrFocusChange)="checkFocusVisible()">
             <!--tab links-->
             <ng-container *ngFor="let link of tabLinkDirectives">
-                <ng-container *ngIf="link.tabsId === tabsId && !link.inOverflow"
-                              [ngTemplateOutlet]="link.templateRefContainer.template">
+                <ng-container *ngIf="link.tabsId === tabsId && !link.inOverflow">
+                    <li role="presentation" class="nav-item">
```

Comment:
Probably this `nav-item` class may be responsible for the tab-width changes we see in the gemini screenshots.
I'd like to revert this resizing for V1 as it would be a visual breaking change that may lead to problems in the layouts of products that update to the next V1.